src/helpers/util.ts

```diff
@@ -1,5 +1,7 @@
 import childprocess from 'child_process'
 
+export const SPAWNPROCESSBUFFERSIZE = 10485760 // 10 MiB
```

Comment:
```suggestion
export const SPAWNPROCESSBUFFERSIZE = 1_048_576 * 10 // 10 MiB
```

(Readability) Underscores can improve readability in place of would-be commas.

Feel free to resolve this suggestion if you feel it does not improve readability.

Reply:
@mitchell-codecov I think it's fine either way (especially because it's commented with 10MiB)